modules/tutorials/pages/pattern-matching/multiple-hop-and-accumulation.adoc

```diff
@@ -153,7 +153,7 @@ WHERE x1.age>x2.age AND x2.@cnt!=x4.@cnt AND (x1.salary+x3.salary)<x4.salary
 GSQL's pattern matching syntax provides the essentials for a regular expression language for paths in graphs.  Consider the three basic requirements for a regular expression language:
 
 . *The empty set* --> A path of length zero (no match)
-. *Concatenation* --> Form a path by adding one on two another. You can write an N-hop pattern, and M-hop pattern, and then combine them to have a (N+M)-hop pattern.
+. *Concatenation* --> Combine paths together. You can write an N-hop pattern, and M-hop pattern, and then combine them to have an (N+M)-hop pattern.
```

Comment:
Concatenate is more precise than the combine: "Form a longer path by chaining one path after another one."